guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -0,0 +1,15 @@
+Adam Růžička <adamruzicka@users.noreply.github.com>
+Bernhard Suttner <sbernhard@users.noreply.github.com>
+Chris Roberts <chrobert@redhat.com>
+Evgeni Golov <evgeni@golov.de>
+Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
+Ian Ballou <ianballou67@gmail.com>
+Jeremy Lenz <jlenz@redhat.com>
+Leos Stejskal <github@stejskalleos.cz>
+Lucy Fu <lufu@redhat.com>
+Maria <mariaaga@redhat.com>
+Partha Aji <paji@redhat.com>
+Pavel Moravec <pmoravec@redhat.com>
+Samir Jha <sjha4@ncsu.edu>
+William Bradford Clark <wclark@redhat.com>
+hao-yu <hyu@redhat.com>
```

Comment:
```suggestion
Hao Yu <hyu@redhat.com>
```
I am not sure whether this name is not capitalized intentionally so pointing it out just in case it was overlooked.

Reply:
Fixed.